### ISSUE TITLE
Parser won't accept a Base 16 hex literal with a lower case b

### ISSUE THREAD

**Issue by ffaristocrat** [godotengine/godot#32963]
**Godot version:**
Godot Engine v3.2.alpha.custom_build.3e782c78a - https://godotengine.org
OpenGL ES 3.0 Renderer: AMD Radeon R9 M370X OpenGL Engine

**OS/device including version:**
macOS Mojave 10.14.6 

**Issue description:**

Including a lower case b returns this error in the editor and when you run.

`Parser Error: Parse error: Invalid numeric constant at 'b'`

Changing it to an upper case b removes the error.

**Steps to reproduce:**

Assign a hex integer with a lower case b to a variable.

**Minimal reproduction project:**

New Project
Add Node2D
Save as main
Create new script, main.gd

```
extends Node2D

var x = 0xabcdef

```
Save and run to get the error.

If you change it to

```
extends Node2D

var x = 0xaBcdef

```

The project will successfully run.

**Comment by ffaristocrat** [comment]
It looks like this commit to add support for binary literals is the culprit.

https://github.com/godotengine/godot/commit/d3cc9c0bf191491ea3beaa47c7ba46dcd0af38e2

**Comment by akien-mga** [comment]
CC @JellyWX @bojidar-bg

**Comment by JellyWX** [comment]
Line 910 in `gdscript_tokenizer.cpp` I believe. The block just needs a new conditional if already within a hexadecimal literal

**Comment by ffaristocrat** [comment]
Created a PR to fix this #32966
